apps/flame-defi/app/swap/hooks/use-swap-button.tsx

```diff
@@ -41,37 +40,29 @@ interface ErrorWithMessage {
 
 const useCheckTokenApproval = (
   topToken: TokenInputState,
```

Comment:
don't need to keep the name `topToken` when there isn't also a `bottomToken`

Reply:
I think you should use the name `tokenInput` actually, because it's not just a `Token`